### PR TITLE
check for spaces and wrap in quotes

### PR DESCRIPTION
**Pull request by DrogoNevets**
This fixes an issue we are having using this with windows
also fixes for the user in issue 139

### CODE REVIEW

File: src/util/MongoBinary.js [hunk 1]
```diff
@@ -155,6 +155,10 @@ export default class MongoBinary {
     if (systemBinary) {
       binaryPath = await this.getSystemPath(systemBinary);
       if (binaryPath) {
+        if(~binaryPath.indexOf(' ')) {
```
Comment (nodkz) [inline src/util/MongoBinary.js]: Please add `// eslint-disable-line o-bitwise`

And run lint before committing:
`npm run lint`

**Comment by DrogoNevets** [comment]
how do i get this to re run?

**Comment by DrogoNevets** [comment]
@nodkz there you go, sorry about that - ive not used prettier before, just let vsc do it for me. (which caused conflicts)

will look into it though, likes like a good way to enforce coding style

**Comment by nodkz** [comment]
Thanks a lot! 👍

**Pull request merged**